tools/make-deb.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+#
+# This script expects to run on Ubuntu. It installs the dependencies necessary
+# to build Boulder and produce a Debian Package. The actual build and packaging
+# is handled by a call to Make.
+#
+
+# -e Stops execution in the instance of a command or pipeline error.
+# -u Treat unset variables as an error and exit immediately.
+set -eu
+
+#
+# Setup Dependencies
+#
+
+sudo apt-get install -y --no-install-recommends \
+  ruby \
+  ruby-dev \
+  gcc
+
+# Parse our production Go version from our docker-compose file.
+GO_VERSION=$(grep "BOULDER_TOOLS_TAG:-" docker-compose.yml | sed -e 's/.*-go//' -e 's/_.*//')
+
+# Download and unpack our production go version.
+$(dirname -- "${0}")/fetch-and-verify-go.sh "${GO_VERSION}"
+sudo tar -C /usr/local -xzf go.tar.gz
+export PATH=/usr/local/go/bin:$PATH
+
+# Install fpm, this is used in our Makefile to package Boulder as a deb or rpm.
+sudo gem install --no-document fpm
```

Comment:
I overlooked this in our Boulder releaser (and will revisit) but we should install a specific fpm version.

Reply:
https://github.com/letsencrypt/boulder/pull/6049